hwinfo/pci/tests/test_lspci.py

```diff
@@ -43,12 +43,12 @@ def test_pci_device_class_name(self):
 
     def test_pci_device_sub_string(self):
         rec = self.parser.parse_items().pop()
-	self._assert_rec_key(rec, 'pci_device_sub_string')
-
+        self._assert_rec_key(rec, 'pci_device_sub_string')
+	    
```

Comment:
You appear to have introduced trailing whitespace here, and in the next hunk too.